<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagementulPerformantei</class>
 <widget class="QMainWindow" name="ManagementulPerformantei">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManagementulPerformantei</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Autentificare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="despreButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Despre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Iesire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>240</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Autentificare</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="passEdit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="userlabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nume utilizator:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passlabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parola:</string>
    </property>
   </widget>
   <widget class="QLabel" name="about1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>941</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Aplicatia are rolul de a gestiona performanta unei companii si implicit a tutror agentiilor acesteia.</string>
    </property>
   </widget>
   <widget class="QLabel" name="about2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>941</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Aceasta are atat rolul de a gestiona performanta cotidiana (obiective,procese etc.), cat si evaluarea companiei prin intermediul unor indicatori.</string>
    </property>
   </widget>
   <widget class="QLabel" name="about3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dezvoltatori:</string>
    </property>
   </widget>
   <widget class="QLabel" name="about4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Oprin Andrei Daniel</string>
    </property>
   </widget>
   <widget class="QLabel" name="about5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dragomir Iulian Valentin</string>
    </property>
   </widget>
   <widget class="QLabel" name="about6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pavel Cristina </string>
    </property>
   </widget>
   <widget class="QLabel" name="about7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>600</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chiriac Ecaterina</string>
    </property>
   </widget>
   <widget class="QLabel" name="about8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>640</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mihalcea Alexandra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inapoiButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>320</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Inapoi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>280</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Inregistreaza companie</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
